sgxs-tools/src/sgx_detect/main.rs

```diff
@@ -308,8 +358,17 @@ fn main() {
         (@arg EXPORT:   --export                                                "Export detected support information as YAML")
         (@arg PLAIN:    --plaintext                                             "Disable color and UTF-8 output")
         (@arg VERBOSE:  --verbose -v                                            "Print extra information when encountering issues")
+        (@arg ENCLAVEOS_PRE:  --enclaveos_pre_install                           "Run extra diagnostics tests for enclaveos pre installation")
+        (@arg ENCLAVEOS_POST: --enclaveos_post_install                          "Run extra diagnostics tests for enclaveos post installation")
```

Comment:
I think dashes are more commonly used for command-line arguments. I think you can specify them like so in this macro: `--("enclaveos-post-install")`

Reply:
@jethrogb  I don't really understand this. Do you suggest that I should replace 

```
(@arg ENCLAVEOS_PRE:  --enclaveos_pre_install
``` 

with 

``` 
(@arg ENCLAVEOS_POST: --("enclaveos-pre-install")
```

?

Reply:
yes